api/src/main/java/org/openmrs/validator/DrugOrderValidator.java

```diff
@@ -93,58 +97,64 @@ public void validate(Object obj, Errors errors) {
 			if (order.getDrug() == null || order.getDrug().getConcept() == null) {
 				ValidationUtils.rejectIfEmpty(errors, "concept", "error.null");
 			}
-			
+
 			if (order.getConcept() != null && order.getDrug() != null && order.getDrug().getConcept() != null
-			        && !order.getDrug().getConcept().equals(order.getConcept())) {
-				errors.rejectValue("drug", "error.general");
-				errors.rejectValue("concept", "error.concept");
-			}
+					&& !order.getDrug().getConcept().equals(order.getConcept())) {
+             errors.rejectValue("drug", "error.general");
+             errors.rejectValue("concept", "error.concept");
```

Comment:
These should be indented with tabs, not spaces. Your IDE might need to be reconfigured to do this.